### DISCUSSION TITLE
📝 README.md translations 🌍

### DISCUSSION THREAD

**Discussion by oleksis** [Textualize/rich#1174] [General]
Authors of translations will be notified in this discussion. If you are an **Author** or **Collaborator** and want to participate, leave a comment in this discussion.

**Comment by Adilius** [comment]
Hejsan! 👋
I write & keep the Swedish readme up-to-date.

**Comment by willmcgugan** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝

**Reply by oleksis** [reply]
Updated the `README.es.md` 🤓

**Reply by oleksis** [reply]
I tested the workflow to Add the commit to check the changes, [here the Test](https://github.com/oleksis/rich/actions/runs/783565224) 🧪✅

**Comment by willmcgugan** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝. Check the [commit 709613e](https://github.com/willmcgugan/rich/commit/709613e4f23f4853f7ddc7a9fb0401f03ec24503) 👀

**Comment by oleksis** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝. Check the [commit 709613e](https://github.com/willmcgugan/rich/commit/709613e4f23f4853f7ddc7a9fb0401f03ec24503) 👀

**Comment by willmcgugan** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝. Check the [commit d39643d](https://github.com/willmcgugan/rich/commit/d39643d182b95fdde75d8f937afc9d9c3a5a4518) 👀

**Comment by willmcgugan** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝. Check the [commit 014d8fa](https://github.com/willmcgugan/rich/commit/014d8fa9c1121a0c7f837309001549a564c5bb1f) 👀

**Comment by willmcgugan** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝. Check the [commit be78d80](https://github.com/willmcgugan/rich/commit/be78d80d91f7f8ad19bc188547c3feee23f51925) 👀

**Comment by Adilius** [comment]
Just an idea, is there a better way to store the READMEs in different languages as to not clutter the root repository folder?
Also another idea, perhaps a github workflow running python script to check if READMEs formatting/links is persistant across other languages.

**Comment by willmcgugan** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝. Check the [commit 13feaae](https://github.com/willmcgugan/rich/commit/13feaae4aee21a922271e7f70dc64a83952e594b) 👀

**Comment by willmcgugan** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝. Check the [commit 8d52f96](https://github.com/willmcgugan/rich/commit/8d52f96667e515ee98afd0b25bdb57e1f0ed2e7d) 👀

**Comment by kilacoda-old** [comment]
नमस्ते! (Namaste/Hello!) 👋
 I wrote the Hindi translation, and would like to be notified to keep it up to date.

**Reply by willmcgugan** [reply]
Namaste! I think you just need to hit the Subscribe button (top right somewhere)

**Reply by kilacoda-old** [reply]
Oh, okay! Thought I needed to write a comment as written in the OP for the Action to detect it 😅

**Reply by oleksis** [reply]
Thanks @kilacoda I will add you to the GitHub Actions if you planning keep updating the Hindi REAMDE.

Namaste 🙏🏾

**Reply by luigibertaco** [reply]
Hey @oleksis happy to be added to the authors for notifications. I just wrote the Brazilian portuguese translation, and keen to keep it updated.

**Reply by oleksis** [reply]
@luigibertaco Obrigado! Get the note 📝 to add 🙏🏾

**Reply by luigibertaco** [reply]
Actually, one can always open a PR to add themselves to the automation workflow. Should I do that instead?

**Comment by willmcgugan** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝. Check the [commit 1b5ab4b](https://github.com/willmcgugan/rich/commit/1b5ab4b46c4ebd1211b6ca0ef8f484ccf2b31aa2) 👀

**Reply by oleksis** [reply]
If you wish to update the Readme workflow for participants in the discussion, they would be added manual or by a GraphQL query?

**Comment by willmcgugan** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝. Check the [commit bf7c38b](https://github.com/willmcgugan/rich/commit/bf7c38ba6368ffd2f188b16e8431f6fb623b08ee) 👀

**Comment by willmcgugan** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝. Check the [commit edec73c](https://github.com/willmcgugan/rich/commit/edec73c074cdaa2428c8d5bead6c18b1ea1ed119) 👀

**Comment by willmcgugan** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝. Check the [commit 473eac5](https://github.com/willmcgugan/rich/commit/473eac577f42a243404c3e01ba13ba73c7aaaadd) 👀

**Comment by willmcgugan** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝. Check the [commit 8cfd988](https://github.com/willmcgugan/rich/commit/8cfd988cae6af85d2210ceb4d83ba770111a7cc4) 👀

**Comment by willmcgugan** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝. Check the [commit 1897318](https://github.com/willmcgugan/rich/commit/189731826bf64e084c9fd642321b9fb1a3177a94) 👀

**Comment by willmcgugan** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝. Check the [commit a1c76b2](https://github.com/willmcgugan/rich/commit/a1c76b2168fc62c3380e989ee21f01bfb5dc0ed2) 👀

**Comment by willmcgugan** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝. Check the [commit 972c43e](https://github.com/willmcgugan/rich/commit/972c43ed32d1c7acaf23d442757935bf0d914203) 👀

**Comment by willmcgugan** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝. Check the [commit 880693d](https://github.com/willmcgugan/rich/commit/880693db5b6b5a33c9bdcad35b57afcf37818860) 👀

**Comment by willmcgugan** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝. Check the [commit e21ac11](https://github.com/willmcgugan/rich/commit/e21ac11a835bb3dc2e6494f18adad13c76b61c71) 👀

**Comment by willmcgugan** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝. Check the [commit e7de32a](https://github.com/willmcgugan/rich/commit/e7de32a0e50026695166b2b397114117ec549cd0) 👀

**Comment by willmcgugan** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝. Check the [commit 1daa177](https://github.com/willmcgugan/rich/commit/1daa1771f31dee625054c65926eb616cc1bc352e) 👀

**Comment by willmcgugan** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝. Check the [commit e4ba549](https://github.com/willmcgugan/rich/commit/e4ba54989c65af9cfeeb3f964a8078305ed991bd) 👀

**Comment by willmcgugan** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝. Check the [commit 9e46a97](https://github.com/willmcgugan/rich/commit/9e46a9729c59189274b7dc8e09179a3ef5ba233b) 👀

**Comment by willmcgugan** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝. Check the [commit 857e0a0](https://github.com/willmcgugan/rich/commit/857e0a0d7be4827746fe42e2bae7db920aecc642) 👀

**Comment by willmcgugan** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝. Check the [commit 19601d3](https://github.com/willmcgugan/rich/commit/19601d3710273c54e0dfb599779c8117327a8cd8) 👀

**Comment by willmcgugan** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝. Check the [commit fabbf40](https://github.com/willmcgugan/rich/commit/fabbf40002fc837d180eae22c11cf8bd07dd3b33) 👀

**Comment by willmcgugan** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝. Check the [commit 695256c](https://github.com/willmcgugan/rich/commit/695256ce5316482f15818962ba8f692a460f923f) 👀

**Comment by willmcgugan** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝. Check the [commit dbd7135](https://github.com/willmcgugan/rich/commit/dbd7135d55e20d6494f4405f25176d37df68ed04) 👀

**Comment by willmcgugan** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝. Check the [commit 8b4f352](https://github.com/willmcgugan/rich/commit/8b4f352c43a969a0ab680ece199224bb97549d14) 👀

**Comment by willmcgugan** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝. Check the [commit 234d1ea](https://github.com/willmcgugan/rich/commit/234d1eaa2a430c7a5766890e3bec13befca6dbc4) 👀

**Comment by willmcgugan** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝. Check the [commit 2b026d8](https://github.com/willmcgugan/rich/commit/2b026d8b960e1ace35b138215094a07db8bbc058) 👀

**Comment by willmcgugan** [comment]
🤓 @willmcgugan @oleksis @Adilius README.md changed 📝. Check the [commit 7135087](https://github.com/willmcgugan/rich/commit/71350877d3459fdfa5805061878db8542785f3a6) 👀
